### PR TITLE
[doc](auth)auth doc

### PR DESCRIPTION
**Pull request by zddr**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

auth doc

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/admin-manual/privilege-ldap/user-privilege.md [hunk 1]
```diff
@@ -50,18 +50,60 @@ Doris 新的权限管理系统参照了 Mysql 的权限管理机制，做到了
...
+
+当用户和角色取消关联，用户自动失去角色的所有权限。
+
+当角色的权限被增加或删除，用户的权限也会随之变更。
+
+```
+                  ┌────────┐         ┌────────┐
+         ┌────┬───►  role1 ├────┬────►  priv1 │
+┌────────┤    │   └────────┘    │    └────────┘
+│  user1 │    │                 │
```
Comment (morningman) [inline docs/zh-CN/docs/admin-manual/privilege-ldap/user-privilege.md:66]: 第一行对齐？

**Comment by zddr** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Comment by zddr** [comment]
run p0

**Pull request merged**